src/urbit/app/chess.hoon

```diff
@@ -793,7 +803,7 @@
           ?~  p.sign
           ::  
             :-  :~  :*  %give  %fact   ~[/challenges]
-                        %chess-update  !>([%challenge-sent src.bowl u.+:(~(get by challenges-sent) src.bowl)])
+                        %chess-update  !>([%challenge-sent src.bowl (~(got by challenges-sent) src.bowl)])
                     ==
                 ==
             this
```

Comment:
See primary comment for this review. Line `833` should now be `%challenge-replied`, and the wire on lines `180`/`826` should now be `/poke/challenge/reply`/`[%poke %challenge %reply ~]`.